packages/map-template/package.json

```diff
@@ -7,7 +7,9 @@
     "dist/*.js",
     "dist/*.mjs"
   ],
-  "dependencies": {},
+  "dependencies": {
+    "uuid": "^9.0.1"
```

Comment:
Should this be moved to devDependencies?

Reply:
✅